src/main/resources/application-dev.yml

```
@@ -14,6 +14,8 @@ services:
     url: "http://localhost:5000/fhir"
   fhirServer:
     url: "http://localhost:8082/fhir"
+    username: ""
```

hier fehlt noch das analog wie in application.yaml einzurücken:

```suggestion
    auth:
      basic:
        enabled: false
        username: ""
        password: ""
```